internal/resources/cloud/resource_synthetic_monitoring_installation.go

```diff
@@ -44,7 +44,6 @@ This resource cannot be imported but it can be used on an existing Synthetic Mon
 `,
 		CreateContext: ResourceInstallationCreate,
 		ReadContext:   ResourceInstallationRead,
-		UpdateContext: ResourceInstallationRead,
```

Comment:
Since all fields are `ForceNew`, the TF framework told me to drop the update func

Reply:
Makes sense.